toolkit/resources/imageconfigs/additionalfiles/iso_initrd/root/runliveinstaller

```diff
@@ -36,6 +40,26 @@ LABEL=CDROM
 
 if grep -qs $ISO_ROOT /proc/mounts; then
     echo ISO root already mounted
+# Read from /proc/cmdline to get image configs from PXE server
+elif [[ $(grep "config_root" $CMDLINE) ]]; then
```

Comment:
Please use `-q` to avoid printing the output - no need to see it.

Also, I think this might be evaluated to a false statement, if "config_root" is inside the output from `/proc/cmdline`, since `[[ ... ]]` evaluates to false for a zero. If I understand your intentions correctly, this might work better:

```suggestion
elif grep -q "config_root" $CMDLINE; then
```